teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -447,12 +462,41 @@ private OptionalInt getP2pLowerBound() {
   }
 
   private OptionalInt getP2pUpperBound() {
-    if (p2pUpperBound.isPresent() && p2pLowerBound.isPresent()) {
-      return p2pLowerBound.getAsInt() > p2pUpperBound.getAsInt() ? p2pLowerBound : p2pUpperBound;
-    }
```

Comment:
I'll put this back in place, we're just smart defaulting the values and we don't want to half remove that.